profile/README.md

```diff
@@ -0,0 +1,27 @@
+<p align="center">
+    <img width="400px" src="https://raw.githubusercontent.com/FluxML/Flux.jl/master/docs/src/assets/logo.png"/>
+</p>
+
+# The Elegant Machine Learning Stack
+
+Flux is a 100% pure-Julia stack and provides lightweight abstractions on top of Julia's native GPU and AD support. It makes the easy things easy while remaining fully hackable.
+
+## Features
+
+Flux has features that sets it apart among ML systems
```

Comment:
Dangling sentence? Maybe something got lost in a diff.

Reply:
Ah, missed a full stop. This was taken from the website, should this be removed though?

Reply:
I see. Just the period should be fine.